### PR TITLE
[Big Team Science Hackathon] Adding to the Research Infrastructure Roles list of job titles

### PR DESCRIPTION
**Pull request by Arielle-Bennett**
### Summary

Part of #2684 and #2685 - we are proposing to live update the list of example role titles in the landing for the Research Infrastructure Roles chapter. This will be facilitated by soliciting additional suggestions from the participants in the Big Team Science _Turing Way_ hackathon on Thurs 27 October 2022. 

### List of changes proposed in this PR (pull-request)

* Adding an expanded list of RIR titles from audience input

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [X] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by Arielle-Bennett** [comment]
**Actual preview:** https://deploy-preview-2750--the-turing-way.netlify.app/collaboration/research-infrastructure-roles.html

File: book/website/collaboration/research-infrastructure-roles.md [hunk 1]
```diff
@@ -32,3 +30,21 @@ But this work is often hidden and not visible in academia, where papers and cita
 The motivation for writing this chapter and providing examples of the type of work that goes into research infrastructure is to make this hidden labour visible to a wider audience and highlight just how critical these aspects of research are for any community or project. 
 
 Other initiatives such as the [Software Sustainability Institute](https://www.software.ac.uk/), the [Center for Scientific Collaboration & Community Engagement](https://www.cscce.org/), and the [Hidden REF](https://hidden-ref.org/), are all contributing to the growing recognition of the importance of these roles for research. 
+
+(cl-research-infrastructure-roles-list)=
+## Example Role Title List
```
Comment (dingaaling) [inline book/website/collaboration/research-infrastructure-roles.md:35]: @Arielle-Bennett Thoughts on renaming this to:

"List of Research Infrastructure Roles" or "Examples of Research Infrastructure Role"

The existing title in the header is somewhat confusing to me

**Review by dingaaling — APPROVED** [review]
Besides potentially rephrasing the header title before the list, lgtm!

File: book/website/collaboration/research-infrastructure-roles.md [hunk 2]
```diff
@@ -32,3 +30,21 @@ But this work is often hidden and not visible in academia, where papers and cita
...
+
+This initial list of examples of Research Infrastructure Roles that has been expanded as part of _The Turing Way_ Hackathon Workshop at Big Team Science conference 2022.
```
Comment (dingaaling) [inline book/website/collaboration/research-infrastructure-roles.md:37]: Shall we add the link to the conf? https://bigteamscienceconference.github.io/

**Review by aleesteele — APPROVED** [review]

**Comment by Arielle-Bennett** [comment]
@all-contributors 
please add @grczh for content 
please add @MamsMandrill for content

**Comment by aleesteele** [comment]
@all-contributors please add @mahmoud-elsherif for content

**Pull request merged**